app_core.py

```diff
@@ -14,8 +14,10 @@
 
 if os.getenv("TESTNET"):
     app.config["TESTNET"] = True
+    app.config["wavesexplorer"] = 'https://testnet.wavesexplorer.com'
```

Comment:
make the config setting upper case like all the other ones

Reply:
wavesexplorer is now in CAPS.